test/e2e/reconciler/cache/replication_test.go

```diff
@@ -60,109 +61,264 @@ type testScenario struct {
 // scenarios all test scenarios that will be run against in-process and standalone cache server
 var scenarios = []testScenario{
 	{"TestReplicateAPIExport", replicateAPIExportScenario},
+	{"TestReplicateAPIResourceSchema", replicateAPIResourceSchemaScenario},
```

Comment:
you don't need to format this as a Go test name, much more common to have e.g. `"replicate APIResourceSchema"` which will then turn into `TestCacheServerInProcess/replicate_APIResourceSchema`